addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -240,10 +240,12 @@ def get(replenishment, key, check_in_receipt=False):
         reserved_quantity = self._get_reserved_qty(move, warehouse, replenish_data)
         missing_quantity = move.product_uom_qty - reserved_quantity
         free_qty = product.uom_id._compute_quantity(product.free_qty, move.product_uom)
-        if not has_to_order_line and float_compare(missing_quantity, free_qty, precision_rounding=move.product_uom.rounding) <= 0:
+        if not has_to_order_line \
+           and float_compare(missing_quantity, 0.1, 2) <= 0 \
+           or float_compare(missing_quantity, free_qty, precision_rounding=move.product_uom.rounding) <= 0:
             return self._format_receipt_date('available')
 
-        max_date = max(map(lambda rep: get(rep, 'date', True), replenishments))
+        max_date = max(map(lambda rep: get(rep, 'date', True), replenishments)) if replenishments else fields.datetime.today()
```

Comment:
`max()` has a `default` argument you can use for this purpose.

Reply:
Ok :slightly_smiling_face: